tests/improved-sanity-test/main.yml

```diff
@@ -210,6 +210,24 @@
       tags:
         - atomic_host_check
 
+    # we remove any subscriptions after the upgrade to verify that
+    # 'rpm-ostree status' with the 'unconfigured-state' field present.
+    # https://bugzilla.redhat.com/show_bug.cgi?id=1421867
+    - role: redhat_unsubscribe
+      unconfigured_state: true
+      when: ansible_distribution == 'RedHat'
+      tags:
+        - redhat_unsubscribe
+
```

Comment:
Do you suggest stopping the daemon as part of the role?  

The daemon will automatically start when a user runs an `rpm-ostree` command, correct?